modules/ROOT/pages/testing-writing-your-first-munit-test.adoc

```diff
@@ -0,0 +1,201 @@
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:basic="http://www.mulesoft.org/schema/mule/basic"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/munit-tools http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+      http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+          http://www.mulesoft.org/schema/mule/basic http://www.mulesoft.org/schema/mule/basic/current/mule-basic.xsd">
+
+    <basic:config name="config" configId="configId">
+        <basic:connection requiredParameter="aValue" />
+    </basic:config>
+
+    <munit:config name="basic-operations-test"/>
+
+    <munit:test name="sayHiTest">
+        <munit:execution>
+            <basic:say-hi person="Your Name"/>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-equals actual="#[payload]" expected="#['Hello!']"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="retrieveInfoTest">
+        <munit:execution>
+            <basic:retrieve-info config-ref="config"/>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-equals actual="#[payload]" expected="#['Using Configuration [configId] with Connection id [aValue:100]']"/>
+        </munit:validation>
+    </munit:test>
+
+</mule>
+----
+
+[[run-your-test]]
+== Run Your MUnit Test
+
+Run your test from the command line using Maven:
+
+[source,bash,linenums]
+----
+mvn clean verify
+----
+
+This outputs the result of your test run:
+----
+================================================================================
+=== Running suite: basic-operations-test.xml                                 ===
+================================================================================
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ Running test: sayHiTest                                                      +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ Running test: retrieveInfoTest                                               +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+================================================================================
+= Tests run: 2 - Failed: 0 - Errors: 0 - Skipped: 0 - Time elapsed: 2.55 sec   =
+================================================================================
+----
+
+== See Also
+
+* xref:munit-extensions-maven-plugin-configuration.adoc[Configuring the MUnit Extensions Maven Plugin]
```

Comment:
If you omit the hard-coded link text, it's added at build time automagically and you don't have to maintain it.